models/LoginForm.php

```diff
@@ -93,22 +85,38 @@ function ($attribute) {
             ],
             'rememberMe' => ['rememberMe', 'boolean'],
         ];
+
+        if (!Yii::$app->getModule('user')->debug)
```

Comment:
I thought about it and i disagree. The view file can be used outside of the user management module, for example when overriding the view file. In this case Yii::$app->module is null and we get an error. Referencing to the 'user' module explicitely seems to be the better option.